download.md

```diff
@@ -130,13 +128,16 @@ To improve the speed and load time of your site, build Prebid.js for only the he
 {% assign bidder_pages = site.pages | where: "layout", "bidder" %}
 {% assign module_pages = site.pages | where: "nav_section", "modules" %}
 
+{: .alert.alert-success :}
+Note if you receive an email with a broken link you most likely selected a configuration that is not supported. Verify that each bidder / module is supported in the selected version. 
```

Comment:
Is there some way we can prevent them from selecting an invalid version and module combination?  Seems ideal over expecting them to wait for a broken e-mail.

Reply:
Not easily since some adapters are in 1.x and some are in 0.34.x and some are in both. Without proper tagging in the docs we can't.